includes/Modules/Analytics.php

```diff
@@ -126,6 +126,80 @@ function () {
 			},
 			0
 		);
+
+		add_action(
+			'admin_init',
+			function() {
+				$this->handle_provisioning_callback();
+			}
+		);
+	}
+
+	/**
+	 * Handles the provisioning callback after the user completes the terms of service.
+	 *
+	 * @since 1.0.0
+	 */
+	private function handle_provisioning_callback() {
+		if ( defined( 'WP_CLI' ) && WP_CLI ) {
+			return;
+		}
+
+		$input = $this->context->input();
+
+		if ( $input->filter( INPUT_GET, 'gatoscallback' ) ) {
```

Comment:
Not a big deal, but would be nicer to do the opposite check and return early, to have less indentation below.

Reply:
good suggestion!